helm/kong-app/Chart.yaml

```diff
@@ -1,15 +1,18 @@
 apiVersion: v1
 description: The Cloud-Native Ingress and API-management
 engine: gotpl
-home: https://konghq.com/
-icon: https://s.giantswarm.io/app-icons/1/png/kong-app-light.png
+home: https://github.com/giantswarm/kong-app
+icon: https://s.giantswarm.io/app-icons/kong/1/light.svg
 maintainers:
 - name: hbagdi
   email: harry@konghq.com
 - name: rainest
   email: traines@konghq.com
 name: kong-app
 sources:
-- https://github.com/giantswarm/kong-app/blob/master/README.md
+- https://konghq.com/
 version: 2.2.0
 appVersion: "2.5"
+annotations:
+  application.giantswarm.io/team: team-cabbage
+  application.giantswarm.io/readme: https://raw.githubusercontent.com/giantswarm/kong-app/v[[ .Version ]]/README.md
```

Comment:
```suggestion
  application.giantswarm.io/readme: https://raw.githubusercontent.com/giantswarm/kong-app/v2.3.2/README.md
```

Reply:
Should be removed entirely, app-build-suite overwrites this annotation anyway